shardingsphere-proxy/shardingsphere-proxy-backend/src/main/java/org/apache/shardingsphere/proxy/backend/text/distsql/rql/resource/DataSourceQueryResultSet.java

```diff
@@ -72,18 +90,66 @@ public boolean next() {
     @Override
     public Collection<Object> getRowData() {
         String dataSourceName = dataSourceNames.next();
+        DataSourceProperties dataSourceProperties = dataSourcePropsMap.get(dataSourceName);
+        DataSourcePoolMetaData poolMetaData = DataSourcePoolMetaDataFactory.newInstance(dataSourceProperties.getDataSourceClassName());
+        Map<String, Object> allProperties = getAllProperties(resource.getDataSources().get(dataSourceName));
```

Comment:
It is unnecessary to get all data source properties, display PoolPropertySynonyms is enough for display.